modules/phase_field/include/materials/ConstantAnisotropicMobility.h

```diff
@@ -24,15 +24,14 @@ class ConstantAnisotropicMobilityTempl : public Material
   ConstantAnisotropicMobilityTempl(const InputParameters & parameters);
 
 protected:
-  virtual void computeProperties(){};
-  virtual void initialSetup();
+  virtual void computeQpProperties();
 
   /// raw tensor values as passed in from the input file
-  std::vector<Real> _M_values;
+  std::vector<Real> _mobility_values;
```

Comment:
:smile:

Reply:
The old member name is using a symbol format that is reserved for implementation specific purposes (underscore followed by a capital letter).